core/src/main/codegen/templates/Parser.jj

```diff
@@ -3955,7 +3959,34 @@ SqlCreate SqlCreate() :
     }
 }
 </#if>
-
+<#if parser.execStatementParserMethods?size != 0>
+/**
+ * Parses an EXEC statement
+ */
+SqlNode SqlExec() :
+{
+	final Span s;
+	SqlNode macro;
+	SqlNode source;
+}
+{
+	(
+		<EXEC> 
+		| 
+		<EXECUTE>
+	)
+	macro = CompoundIdentifier() { s = span(); }	
```

Comment:
Should extracting the macro be moved into the SqlExecMacro method in parserImpls.ftl?

Reply:
Hard to say, not sure what other queries will start with EXEC. If there aren't any it can be moved in.

Reply:
From purely reading the function names alone (SqlExec vs. SqlExecMacro), it would seem most fitting to move the macro processing into the latter. That leaves future flexibility for us to handle EXEC/EXECUTE statements that don't have macros in this outer function.

Reply:
Ah that makes sense, will make the change.